js/modules/k6/grpc/client.go

```diff
@@ -373,6 +426,7 @@ type connectParams struct {
 	Timeout               time.Duration
 	MaxReceiveSize        int64
 	MaxSendSize           int64
+	TLSConfig             map[string]interface{}
```

Comment:
Rename this to just `TLS`? `connectParams` is already a configuration object, so `Config` seems redundant.